eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/TcpClient.java

```diff
@@ -104,9 +106,10 @@ public void initChannel(SocketChannel ch) {
         ChannelFuture f = bootstrap.connect(host, port).sync();
         InetSocketAddress localAddress = (InetSocketAddress) f.channel().localAddress();
         channel = f.channel();
-        log
-                .info("connected|local={}:{}|server={}", localAddress.getAddress().getHostAddress(), localAddress.getPort(),
-                        host + ":" + port);
+        if (LOGGER.isInfoEnabled()) {
+            LOGGER.info("connected|local={}:{}|server={}", localAddress.getAddress().getHostAddress(),
+                    localAddress.getPort(), host + ":" + port);
```

Comment:
Hi, may I know why do we need to check log level here, as far as I know the `LOGGER.info`  method has already checked the log level.